packages/quiz/src/styles.css

```diff
@@ -0,0 +1,12 @@
+[data-sr-quiz-submit] {
+  background: rgba(79, 70, 229, 1);
+  color: yellow;
+  border: 4px solid green;
+  &:hover {
+    background: green;
+  }
+}
+[data-sr-quiz-submit][disabled] {
+  cursor: not-allowed;
+  background: gray;
+}
```

Comment:
very much inspired by ReachUI's use of attribute selectors ([example](https://github.com/reach/reach-ui/blob/develop/packages/checkbox/styles.css))

Reply:
i’m a fan of this